chainer/sequential.py

```diff
@@ -0,0 +1,400 @@
+    method in the same order as the given argments.
+    Therefore, you do not need to write the forward pass computation
+    explicitly.
+
+    .. admonition:: Example
+
+        The below example code shows how to use this class to construct a
+        simple sequential network::
+
+          import chainer
+          import chainer.functions as F
+          import link.Links as L
+          from chainer import Sequential
+
+          # Model definition without writing __call__ function
+          model = Sequential(
+              L.Linear(n_in, n_hidden),
+              F.relu,
+              L.Linear(n_hidden, n_hidden),
+              F.relu,
+              L.Linear(n_hidden, n_out)
+          )
+
+          # Compute the forward pass
+          y = model(x)
+
+        where ``x`` denotes a mini-batch of ``n_in``-dimensional input vectors.
+
+        Furthermore, :class:`~Sequential` supports build-in list APIs, so you
+        can concatenate :class:`~Sequential` objects to create a longer
+        :class:`~Sequential` model easily with the same ways as Python lists::
+
+          model_A = Sequential(L.Linear(10, 10), F.relu)
+          model_B = Sequential(L.Linear(10, 10), F.sigmoid)
+          model_C = model_A + model_B
+
+        To repeat a :class:`~Sequential` object multiple times, you can use
+        :meth:`~chainer.Link.repeat` method.
+
+          model_D = model_A.repeat(3)
+
+        You can also add your own functions or any callable objects to a
+        :class:`~Sequential` object::
+
+          from link.Links.model.vision.vgg import VGG16Layers()
+
+          model = Sequential()
+          model.append(L.Linear(n_out, n_hidden))
+          model.append(F.relu)
+          model.append(F.Reshape((1, 3, 224, 224)))
+          model.append(VGG16Layers())
+          model.append(lambda x: x['prob'])
+
+          y = model(x)
+
+        The above code example shows how to add some layers to the ``model``
+        using :meth:`~Sequential.append` method and then add a large network
+        (``VGG16Layers``) and finally add a lambda function to extract the
+        ``prob`` output.
+
+        Note that a :class:`~Sequential` link which has at least one ``lmabda``
+        function as its member cannot be pickled. So, please use ``partial``
+        method from ``functools`` package instead::
+
+          from functools import partial
+
+          # This is not pickable
+          model = Sequential(
+              L.Convolution2D(None, 64, 3, 1, 1),
+              lambda x: F.max_pooling_2d(x, 2)
+          )
+
+          # This is pickable
+          model = Sequential(
+              L.Convolution2D(None, 64, 3, 1, 1),
+              partial(F.max_pooling_2d, ksize=2)
+          )
+
+    Args:
+        layers: The layers which are called in its order. Each component should
+            be a callable object including :class:`~Link` object and
+            :class:`~Function` class.
```

Comment:
I don't think it's good to mention `Function` class here because it is not recommended to reuse the same `Function` object multiple times. It's ok to mention the functions defined under the `chainer.functions` namespace, e.g. `F.relu`.

Reply:
Thanks. I modified this part to:

```
        layers: The layers which are called in its order. Each component should
            be a callable object including :class:`~Link` object and
            functions defined under the :mod:`chainer.functions`, e.g.,
            :obj:`~chainer.functions.relu`, etc.
```